definitions/divorce/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -9789,864 +9789,941 @@
     "ShowSummaryChangeOption": "No"
   },
   {
-    "LiveFrom": "07/04/2020",
+    "LiveFrom": "20/09/2019",
     "CaseTypeID": "DIVORCE",
-    "CaseEventID": "solicitorUpdate",
-    "CaseFieldID": "LabelSolAboutEditingApplication-LegalProceedings",
-    "PageFieldDisplayOrder": 0,
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "D8PetitionerLastName",
+    "PageFieldDisplayOrder": 99,
     "DisplayContext": "READONLY",
-    "PageID": "SolExistingCourtCasesUpdate",
-    "PageLabel": "Other legal proceedings",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
     "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "ReceivedAOSfromRespDate=\"nevershow\"",
     "ShowSummaryChangeOption": "No"
   },
   {
-    "LiveFrom": "07/04/2020",
+    "LiveFrom": "20/09/2019",
     "CaseTypeID": "DIVORCE",
-    "CaseEventID": "solicitorUpdate",
-    "CaseFieldID": "LabelSolAboutEditingApplication-FinancialOrders",
-    "PageFieldDisplayOrder": 0,
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "D8RespondentFirstName",
+    "PageFieldDisplayOrder": 99,
     "DisplayContext": "READONLY",
-    "PageID": "SolDividingMoneyAndPropertyUpdate",
-    "PageLabel": "Financial orders",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
     "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "ReceivedAOSfromRespDate=\"nevershow\"",
     "ShowSummaryChangeOption": "No"
   },
   {
-    "LiveFrom": "07/04/2020",
+    "LiveFrom": "20/09/2019",
     "CaseTypeID": "DIVORCE",
-    "CaseEventID": "solicitorUpdate",
-    "CaseFieldID": "LabelSolAboutEditingApplication-ClaimCosts",
-    "PageFieldDisplayOrder": 0,
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "D8RespondentLastName",
+    "PageFieldDisplayOrder": 99,
     "DisplayContext": "READONLY",
-    "PageID": "SolApplyToClaimCostsUpdate",
-    "PageLabel": "Claim for costs",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
     "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "ReceivedAOSfromRespDate=\"nevershow\"",
     "ShowSummaryChangeOption": "No"
   },
   {
-    "LiveFrom": "07/04/2020",
+    "LiveFrom": "20/09/2019",
     "CaseTypeID": "DIVORCE",
-    "CaseEventID": "solicitorUpdate",
-    "CaseFieldID": "LabelSolAboutEditingApplication-UploadMarriageCert",
-    "PageFieldDisplayOrder": 0,
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "D8ReasonForDivorce",
+    "PageFieldDisplayOrder": 99,
     "DisplayContext": "READONLY",
-    "PageID": "SolUploadDocsUpdate",
-    "PageLabel": "Upload the marriage certificate",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
     "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "ReceivedAOSfromRespDate=\"nevershow\"",
     "ShowSummaryChangeOption": "No"
   }
```

Comment:
I think those (i.e. with display order: 99) were at the beginning of the list for this event.
They display at the top in the event on CCD also, so not sure why the display order is 99... maybe this field doesn't work :D

Reply:
Thanks. Moved them back to start of list. So that any existing login/reason why it's that way is not tampered with.